src/common/views/crud_view.py

```diff
@@ -327,6 +333,10 @@ def show_model_form(
                 model_content_type, instance_obj
             )
 
+            # get nested criteria data if model_name contains rule
+            if "rule" in self.model_name:
```

Comment:
Instead of checking for "rule", how about we just have a list defined for model names that are considered as "rule"s?

We could also define the list in the Django settings.